src/App.js

```diff
@@ -0,0 +1,37 @@
+let todoList = []
+
+todoList = [
+  {
+    id: 1,
+    title: 'Complete assignment 1'
+  },
+  {
+    id: 2,
+    title: 'Complete assignment 2'
+  },
+  {
+    id: 3,
+    title: 'Complete assignment 3'
+  }
+]
+
+function App() {
+  return (
+    <div>
+      <header style={{ textAlign: 'center' }}>
+        <h1>Todo List</h1>
+      </header>
+      <hr />
+
+      <ul>
+        {todoList.map((item) => {
+          return (
+            <li key={item.id}>{item.title}</li>
+          )
+        })}
+      </ul>
+    </div>
+  );
```

Comment:
Looks good! Maybe try an implicit return next time. The code will look cleaner in the long run.
So something like this:

`{todoList.map(item => (<li key={item.id}>
        {item.title}</li>))}`
You see how I'm not calling return explicitly. You don't want to do this all the time but w/e you're only returning one thing, try an implicit return. When we eventually add prettier, prettier will make what i did "prettier" 😆